# ########## trajectory planner params ##########

TrajectoryPlannerROS:
  # DWAPlannerROS:
  acc_lim_x: 0.3
  acc_lim_th: 0.3
  max_vel_x: 0.6
  min_vel_x: 0.0
  max_rot_vel: 0.6
  min_rot_vel: 0.2

  yaw_goal_tolerance: 0.1
  xy_goal_tolerance: 0.3

  sim_time: 3.0
  sim_granularity: 0.025
  vx_samples: 3
  vy_samples: 10
  vtheta_samples: 20
  controller_frequency: 20.0

  path_distance_bias: 32.0
  goal_distance_bias: 24.0
  occdist_scale: 0.01
  forward_point_distance: 0.325
  stop_time_buffer: 0.2
  scaling_speed: 0.25
  max_scaling_factor: 0.2


